CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Change Log
 This file documents all notable changes to juttle-cloudwatch-adapter. The release numbering uses [semantic versioning](http://semver.org).
 
+## 0.4.0
+Released 2016-03-23
+### Minor Changes
+- Update to reflect changes in juttle 0.7.0. This release is compatible with adapter version 0.7.0. [[#13](https://github.com/juttle/juttle-cloudwatch-adapter/pull/13)].
```

Comment:
Same comment as in https://github.com/juttle/juttle-aws-adapter/pull/12#discussion_r57207522